web/body.html

```diff
@@ -0,0 +1,12 @@
+{{ define "body.html" }}
```

Comment:
💭 It looks like these are these just example files? If so, I'm not sure if we should commit them here!

Reply:
Yes, should i move this to an folder example or delete it entirely?

Reply:
@dirien I think it would be okay to remove them and use documentation in the README and on chartmuseum.com to explain how to use the feature

Reply:
ok, @cbuto will remove it. I already have extended the REAMDE. As soon this PR gets merged I will create a PR for the repo holding https://github.com/chartmuseum/www

Reply:
done